src/index.js

```diff
@@ -1,113 +1,109 @@
-const isEnvParamEmpty = ([paramName]) => !process.env[paramName] || process.env[paramName].trim() === ''
-const isMandatory = ([, envParamConfig]) => envParamConfig.mandatory
+const isEnvParamEmpty = ([paramName]) => !process.env[paramName] || process.env[paramName].trim() === '';
+const isMandatory = ([_, envParamConfig]) => envParamConfig.mandatory;
```

Comment:
should be ([name, envParamConfig]) => envParamConfig.mandatory;

Reply:
can't use `name` getting ESLint Error : `ESLint: 'name' is defined but never used.(no-unused-var)`

Reply:
resolve